service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -1702,4 +1703,14 @@ public OrderCancellationReasonDto getOrderCancellationReason(Long orderId) {
             .cancellationComment(order.getCancellationComment())
             .build();
     }
+
+    @Override
+    public ReasonNotTakingBagDto getReasonNotTakingBag(Long orderId) {
+        Order order = orderRepository.findById(orderId)
+            .orElseThrow(() -> new NotFoundException(ORDER_WITH_CURRENT_ID_DOES_NOT_EXIST + orderId));
+        return ReasonNotTakingBagDto.builder()
```

Comment:
should we check if `OrderStatus == NOT_TAKEN_OUT `

Reply:
I don't think it's necessary because we don't care about the status, only the reason not taking bag